### ISSUE TITLE
drive volume name was changed after run build.cmd

### ISSUE THREAD

**Issue by cnblogs-dudu** [dotnet/corefx#564]
After run build.com, I found C: and F: drive volume name had been changed to BCLTest.
![screen shot 2015-01-30 at 10 51 23](https://cloud.githubusercontent.com/assets/3196961/5970442/078adea6-a86e-11e4-9937-7e02b336a81b.png)
Check build log, the following error was found:
```
Starting:    System.IO.FileSystem.DriveInfo.Tests
System.IO.FileSystem.DriveInfoTests.Get_VolumeLabel.Test01 [FAIL]
Error, Wrong volume label returned. Expected: , Returned: BCLTest
Stack Trace:
          e:\Dev\GitHub\DotNet\corefx\src\System.IO.FileSystem.DriveInfo\tests\GetVolumeLabelTests.cs(106,0): 
at System.IO.FileSystem.DriveInfoTests.Get_VolumeLabel.Test01()
Finished:    System.IO.FileSystem.DriveInfo.Tests
```

**Comment by nicholaspei** [comment]
:+1:  园长开始研究这个代码了呀。

**Comment by krwq** [comment]
@nicholaspei, not sure what you meant by that.

If we need to test renaming drive label we should probably at least not do it when running build.cmd and in addition to that change it back to the original name.

**Comment by cnblogs-dudu** [comment]
@krwq 
I can reproduce reproduce this issue. Add a line code `throw new UnauthorizedAccessException();` in GetVolumeLabelTests.cs:
```csharp
for (int i = 0; i < drives.Length; i++)
{
    if (originalVolumeInfo.ContainsKey(drives[i].Name))
    {
        try
        {
            drives[i].VolumeLabel = (String)originalVolumeInfo[drives[i].Name];
            throw new UnauthorizedAccessException();
        }
        catch (UnauthorizedAccessException) { }
    }
```

**Comment by stephentoub** [comment]
Fixed by #1109
